### ISSUE TITLE
CurrencyShop Points are not Per Server

### ISSUE THREAD

**Issue by SwiftSwamp** [cyberpwnn/GlacialRealms#382]
Tested with Vote Points but assuming it's for all Currency's stored within Phantom, if you spend or gain points on one server you have the same amount on every server, it was meant to be per server, I don't know if this was intentionally or not but it's meant to be per server.

**Comment by kotik1998** [comment]
Yup, ive tested it on skyblock and on factions , had the same points

**Comment by cyberpwnn** [comment]
https://github.com/cyberpwnn/CurrencyShops/commit/33ebdd4f508935d09bbb0bccf4384182dadb06f4
